firefox-ios/Client/Frontend/TrackingProtection/TrackingDetailsHelpers/TrackingProtectionStatusView.swift

```diff
@@ -41,7 +41,11 @@ class TrackingProtectionStatusView: UIView {
 
     // MARK: View Setup
     private func setupView() {
-        layer.cornerRadius = TPMenuUX.UX.viewCornerRadius
+        if #available(iOS 26.0, *) {
+            layer.cornerRadius = TPMenuUX.UX.newStyleCornerRadius
```

Comment:
nit: Can we do the checking in the UX structure instead of here and have one single corner radius variable. It cleans a bit nicer